packages/js/src/redux/selectors/facebookEditor.js

```diff
@@ -16,7 +16,7 @@ import {
  * @param {string} value The value to be saved.
  * @returns {string} The value to be saved.
  */
-export const getSocialTitleWithoutTemplate = ( value ) => {
+export const getCustomSocialTitle = ( value ) => {
```

Comment:
I like the previous name more 😅 
I have no hint what custom means without looking at the code